core/validation/failures.py

```diff
@@ -421,6 +423,7 @@ class InvalidOutcomeProjectFailure(ValidationFailure):
 
     invalid_project: str
     section: str
+    index: list[int]
```

Comment:
Maybe just my preferance but prehaps we should keep the naming convention of these indexes the same throughout. Prehaps row_indexes is more descriptive. What do you think?

Reply:
If they already have a standardised name from the R4-specific PR then I'm happy to change to that, I'll take a look in a second. If not, then row_indexes or indexes are both fine.